assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -464,8 +464,12 @@ che.infra.kubernetes.client.http.connection_pool.keep_alive_min=5
 # If not set, every workspace will be created in a new project, where project name = workspace id
 che.infra.openshift.project=
 
-# Create routes with Transport Layer Security (TLS) enabled
-che.infra.openshift.tls_enabled=false
+# Creates Ingresses with Transport Layer Security (TLS) enabled
+# In OpenShift infrastructure, Routes will be TLS-enabled
+che.infra.kubernetes.tls_enabled=false
+
+# Name of a secret that should be used when creating workspace ingresses with TLS
+che.infra.kubernetes.tls_secret=
```

Comment:
Please add a note that it is ignored by OpenShift infrastructure, isn't it?

Reply:
correct. done.